builtin/logical/cassandra/path_roles.go

```diff
@@ -123,11 +128,14 @@ func (b *backend) pathRoleCreate(
 		return logical.ErrorResponse(fmt.Sprintf(
 			"Error parsing lease value of %s: %s", leaseRaw, err)), nil
 	}
+	
+	consistencyStr := data.Get("consistency").(string)	
```

Comment:
Can we verify if the input is parsable before storing it? This way, the value read from the role entry can be used directly.